### PR TITLE
docs: updated readme with project status

### PR DESCRIPTION
**Pull request by ivpavici**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+- [x] RPC v0.4.0
+- [x] [Dump & Load](https://github.com/0xSpaceShard/starknet-devnet-rs#dumping--loading)
+- [x] [Mint token - Local faucet] https://0xspaceshard.github.io/starknet-devnet/docs/guide/mint-token
+- [x] Starknet.js test suite passes 100%
+
+### TODO to reach feature parity with the Pythonic Devnet
+
+- [ ] [Custom accounts implementation](https://0xspaceshard.github.io/starknet-devnet/docs/guide/accounts#custom-implementation)
+  - [ ] [PR](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196)
+- [ ] RPC v0.5.0
```
Comment (FabijanC) [inline README.md:33]: Pythonic Devnet is using 0.3.0

File: README.md [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+  </a>
+</p>
+
+A local testnet for Starknet... in Rust!
+
+This repository is work in progress, please be patient. Please check below the status of features compared with the [Pythonic Devnet](https://github.com/0xSpaceShard/starknet-devnet):
+
+### Supported Features
+
+- [x] `blockifier` version v0.3.0-rc0
```
Comment (FabijanC) [inline README.md:23]: Why would users care about which version of blockifier we are using?

File: README.md [hunk 3]
```diff
@@ -115,6 +147,8 @@ Unlike Pythonic Devnet, which supported the gateway and feeder gateway API, Devn
 
 The JSON-RPC API is reachable via `/rpc` and `/` (e.g. if spawning Devnet with default settings, these URLs have the equivalent functionality: `http://127.0.0.1:5050/rpc` and `http://127.0.0.1:5050/`)
 
+> Out of Starknet **trace** API RPC methods, only `starknet_simulateTransactions` is supported.
```
Comment (FabijanC) [inline README.md:150]: Not sure if `>` is appropriate for this kind of note, it's usually used for [blockquotes](https://www.markdownguide.org/basic-syntax/#blockquotes-1)

File: README.md [hunk 4]
```diff
@@ -1,8 +1,40 @@
...
+
+### Supported Features
+
+- [x] `blockifier` version v0.3.0-rc0
+- [x] RPC v0.4.0
+- [x] [Dump & Load](https://github.com/0xSpaceShard/starknet-devnet-rs#dumping--loading)
+- [x] [Mint token - Local faucet] https://0xspaceshard.github.io/starknet-devnet/docs/guide/mint-token
+- [x] Starknet.js test suite passes 100%
+
+### TODO to reach feature parity with the Pythonic Devnet
```
Comment (FabijanC) [inline README.md:29]: Availability as a package (i.e. crate) is arguably a feature needed for reaching feature parity.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,40 @@
...
 
-A local testnet for Starknet... in Rust
+<h1 align="center" style="margin-top: 12px !important">Starknet Devnet RS</h1>
 
-This repository is work in progress, please be patient. As of Starknet 0.12.2, [Pythonic Devnet](https://github.com/0xSpaceShard/starknet-devnet) is still the way to go.
+<p align="center" dir="auto">
+  <a href="https://hub.docker.com/r/shardlabs/starknet-devnet-rs/tags">
+    <img src="https://img.shields.io/badge/dockerhub-images-important.svg?logo=Docker" style="max-width: 100%;">
+  </a>
+  <a href="https://starkware.co/" rel="nofollow">
```
Comment (FabijanC) [inline README.md:12]: Should we be using this `rel="nofollow"`? My impression is no.

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ivpavici) [reply README.md:23]: just a info for the users, not that important, I can remove it if you think it's better?

Comment (FabijanC) [reply README.md:23]: I think it's better removed.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,44 @@
...
+
+### Supported Features
+
+- [x] `blockifier` version v0.3.0-rc0
+- [x] RPC v0.4.0
+- [x] [Dump & Load](https://github.com/0xSpaceShard/starknet-devnet-rs#dumping--loading)
+- [x] [Mint token - Local faucet](https://0xspaceshard.github.io/starknet-devnet/docs/guide/mint-token)
+- [x] Starknet.js test suite passes 100%
+
+### TODO
```
Comment (FabijanC) [inline README.md:29]: We are currently tracking progress using issues and the PR template reminds us to link closeable. If we are going to introduce a new feature list, we need to remember to edit it when needed, so I propose editing the PR template by introducing a new bullet that will remind us to edit this new TODO section.

**Review by FabijanC — APPROVED** [review]

**Pull request merged**